### DISCUSSION TITLE
运行cli_demo等直接结束，无法对话，Process finished with exit code -1073741819 (0xC0000005)

### DISCUSSION THREAD

**Discussion by jiaxurui** [zai-org/ChatGLM3#414] [Q&A] [answered]
哈喽~尝试运行cli_demo以及web_demo之后，进程直接结束无法进行对话，提示：Process finished with exit code -1073741819 (0xC0000005)“。尝试了很多次没有办法解决，但是换了一台机器后又可以运行。可以请大佬帮我看看嘛，不胜感激！
![2976191d8701a18be0643b79d82f07082938fe99](https://github.com/THUDM/ChatGLM3/assets/145969496/a0fec13f-5a0d-4267-918f-69ca4d020462)
环境：windows10 python3.11 i5-9600K  16GB 4060ti16g

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
很有可能的问题是
1. 你的内存或者显存炸了
2. 你的环境变量没有对，尝试一下直接导入模型，不走环境变量

**Reply by jiaxurui** [reply]
谢谢回复，我去试下~
